src/main/scala/com/fulcrumgenomics/umi/CallMolecularConsensusReads.scala

```diff
@@ -167,7 +161,7 @@ class CallMolecularConsensusReads
       errorRatePreUmi              = errorRatePreUmi,
       errorRatePostUmi             = errorRatePostUmi,
       minInputBaseQuality          = minInputBaseQuality,
-      minConsensusBaseQuality      = minConsensusBaseQuality,
+      minConsensusBaseQuality      = PhredScore.MinValue,
```

Comment:
While you are here I noticed line 127 is not used, and oddly the compiler didn't flag it.